torch/distributed/run.py

```diff
@@ -607,7 +610,24 @@ def get_args_parser() -> ArgumentParser:
         help="torchrun.logs_specs group entrypoint name, value must be type of LogsSpecs. "
         "Can be used to override custom logging behavior.",
     )
-
+    parser.add_argument(
```

Comment:
cc @EikanWang didn't someone from your end already send a PR to do NUMA binding in torchrun? That vaguely rings a bell to me.